daemons/dss-notify/.chalice/config.json

```diff
@@ -10,6 +10,7 @@
     }
   },
   "version": "2.0",
-  "lambda_timeout": 300,
-  "lambda_memory_size": 1536
+  "lambda_timeout": 360,
```

Comment:
Why has the TTL been increased?

Reply:
The indexer's been timing out so I thought it might help.  I can leave it at 300 if you feel it's unnecessary though.